cli/cmd/plugin/unmanaged-cluster/test/e2e/utils/cli.go

```diff
@@ -6,6 +6,7 @@ package e2e
 import (
 	"bytes"
 	"fmt"
+	"github.com/vmware-tanzu/community-edition/cli/cmd/plugin/unmanaged-cluster/tanzu"
```

Comment:
This should be moved down into its own local import group.

Reply:
Hey @stmcginnis Thanks for the review I am fixing the linting issue. Just a min.